Cargo.toml

```diff
@@ -15,6 +19,8 @@ serde_json = "1.0"
 reqwest = { version = "0.10", features = ["json", "stream"] }
 url = "2.1"
 futures = "0.3"
+tokio = { version = "0.2", features = ["macros", "rt-threaded", "io-util", "io-driver"] }
```

Comment:
Is the `io-driver` feature actually required here?
Also please move the `macros` and `rt-threaded` features back to dev-dependencies as they are not actually used in the implementation.